### DISCUSSION TITLE
PayloadTooLargeError: request entity too large

### DISCUSSION THREAD

**Discussion by bertovn** [lowdefy/lowdefy#1601] [Questions and Help]
Hey Lowdefy Team,
I got the error "PayloadTooLargeError: request entity too large" and it's not a news. The strange thing that i have two identical pages (i copied original page and made some changes), where the same requests performed, but one of pages works fine and the second one - works with error from screenshot. Any suggestions?
![image](https://github.com/lowdefy/lowdefy/assets/54134702/c5114595-c5fb-4332-b171-094d360f1511)
![image](https://github.com/lowdefy/lowdefy/assets/54134702/17646960-0793-41a9-9c75-2c8365e79181)

There is no too large response - just array of 9 objects with 5 properties for each of object.
I found topic like this, but i cant start v4 platform because:
![image](https://github.com/lowdefy/lowdefy/assets/54134702/9f3fc553-a3a4-4ecc-b0e3-2ef2623828ca)

**Comment by Gervwyk** [comment]
Hi @bertovn 
This is indeed most likely due to v3 sending full state to the server on request. This often happens on complicated pages with v3. The easiest way to fix it is to reduce the size of state and global on the page. Usually it happens because a large request response is written into state, and can then easily be fixed.
 
This is not always so easy, and for that reason in v4 we've decided that it would be better if the developer explicitly decides what to include in the payload. So in v4, state and global are not included in the request and you can pass the values that you use in the request through the request payload. 

For the running v4 error, please try running `pnpx lowdefy@4.0.0-rc.9 dev`? Important to specifically write the Lowdefy version you want to use. Does this help?
(npm resoves `pnpx lowdefy dev` to the latest stable release which is `3.23.3`, hence the error.)

**Reply by bertovn** [reply]
Hey, dear @Gervwyk and thank you for your help as well :)
Unfortunately solution you've recommended can't be applied in  my case, because i really need this variables and cant avoid them. Also i can't use the request data in raw mode because i build List entity from this list of objects, but after number of experiments i hope that i found solution: errors come in case if i call request and assign variable in Button or ConfirmModal blocks. But if i use Event:onInit in parent - it's OK :)
